src/add-to-app/ios/add-flutter-screen.md

```diff
@@ -690,7 +690,7 @@ FlutterViewController.
 let flutterEngine = FlutterEngine()
 // FlutterDefaultDartEntrypoint is the same as nil, which will run main().
 engine.run(
-  withEntrypoint: FlutterDefaultDartEntrypoint, initialRoute: "/onboarding")
+  withEntrypoint: "main", initialRoute: "/onboarding")
```

Comment:
Im not familiar with this, but just from the comment, FlutterDefaultDartEntrypoint seems to be just a placeholder, if i read it right. Can you try if passing nil works?

Reply:
may also wanna remove the comment above this line

Reply:
actually, i may be terribly wrong - `FlutterDefaultDartEntrypoint` is an actual string that should just work: 

```
If this is FlutterDefaultDartEntrypoint (or nil); this will default to `main()`. 
```

Not sure why `FlutterDefaultDartEntrypoint` fails to work here...